account/account.go

```diff
@@ -533,6 +533,27 @@ func (account *Account) WaitForTransactionReceipt(ctx context.Context, transacti
 	}
 }
 
+// AddTransaction calls dynamically one of Add<type>Transaction methods
```

Comment:
Let's make a more didactic description for better user understanding.
It's a function that can send Invoke, Declare, and Deploy transactions. Explain it your way.

Reply:
Done